### PR TITLE
LPD-26166 Use URL searchParams to check the objectFolderName parameter in the URL avoiding backURL cases

### PR DESCRIPTION
**Pull request by igor-franca**
related [ticket](https://liferay.atlassian.net/browse/LPD-26166)

### CODE REVIEW

**Comment by igor-franca** [comment]
ci:test:sf

**Review by rebsilva — APPROVED** [review]
LGTM! :tada: :cherry_blossom:

File: modules/test/playwright/tests/object-web/objectFolder.spec.ts [hunk 1]
```diff
@@ -91,6 +91,21 @@ test.describe('manage object definitions through model builder', () => {
...
+		modelBuilderPage,
+		page,
+	}) => {
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		await modelBuilderPage.toggleSidebarsButton.click();
+
+		await page.locator('button[data-qa-id="userPersonalMenu"]').click();
+
+		await page.getByRole('menuitem', {name: 'Account Settings'}).click();
```
Comment (larissacribeiro) [inline modules/test/playwright/tests/object-web/objectFolder.spec.ts:105]: Hey @igor-franca, We have an `goToAccountSettings()` method in the AccountSettingsPage, containing these same actions, so you could have this line instead
`await accountSettingsPage.goToAccountSettings();`

However, the locators they are using are a little off and you would have to fix it as well, it's a minor change, I've tested locally and it works. But feel free to do as you see fit!

**Review by larissacribeiro — CHANGES_REQUESTED** [review]
Great work @igor-franca! 🎉 
Just a comment, I know it isn't related to your changes, but could you please move the test `'object definitions from a deleted folder are moved to the default folder'` into the `('manage object definitions through view object definitions') describe`, to keep the tests organized? Thanks!

**Comment by igor-franca** [comment]
ci:test:sf

File: modules/test/playwright/tests/object-web/objectFolder.spec.ts (hunk 1 above)
Comment (igor-franca) [reply modules/test/playwright/tests/object-web/objectFolder.spec.ts]: Hi @larissacribeiro, I decided not to change the `userPersonalMenuButton` locator. I could use `getByTitle('User Profile Menu')`, but the testId approach can be useful when we want to create a localizer that cannot be affected by language changes. I think this is the real reason for this use, even if it doesn't work for our context. 

But I took the opportunity to use the other locators and create a new one on the Account Settings page, thanks!

**Comment by rebsilva** [comment]
great work @igor-franca and great review @larissacribeiro! you rock! :guitar: :sparkles:

**Review by larissacribeiro — APPROVED** [review]
LGTM!

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:playwright

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:forward:force

**Pull request closed without merging**